src/core/acroform/PDFAcroField.ts

```diff
@@ -7,6 +7,20 @@ import PDFNumber from 'src/core/objects/PDFNumber';
 import PDFArray from 'src/core/objects/PDFArray';
 import PDFRef from 'src/core/objects/PDFRef';
 
+const findLastMatch = (value: string, regex: RegExp) => {
```

Comment:
This function is very similar to the following: https://github.com/Hopding/pdf-lib/blob/4be96eb269910bbc6d3ddcd48f2a6382742e179b/src/api/form/appearances.ts#L111-L121

Please move this function into https://github.com/Hopding/pdf-lib/blob/master/src/utils/strings.ts and remove the function defined in `appearances.ts` to avoid duplicate code and logic.